security/monitoring.adoc

```diff
@@ -0,0 +1,60 @@
+[[security-monitoring]]
+= Monitoring cluster events and logs
+{product-author}
+{product-version]
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+[[security-monitoring-intro]]
+== Introduction
+
+In addition to security measures mentioned in other sections of this guide, the ability to monitor and audit an {product-title} cluster is an important part of safeguarding the cluster itself and its users against inappropriate usage.
+
+There are two main sources of cluster-level information that administrators can use: events and logs.
+
+[[security-monitoring-events]]
+== Cluster events
+
+Cluster administrators are encouraged to familiarize themselves with the _Event_ resource type and review a xref:../dev_guide/events.adoc#events-reference[comprehensive list of events]. When monitoring or auditing a cluster, pay special attention to *node* and *system* events.
+
+Events related to resource creation, modification or deletion are good candidates to detect misuse of the cluster.
+Other events may also indicate inappropriate read-only access to protected data.
+
+
+[[security-monitoring-logs]]
+== Cluster logs
+
+Use `*journalctl*`
+
+- API Server logs
+- Kubelet logs
+- Proxy logs
+
+Link back to logging stack
+
+Others: build logs, deployment logs, ...
+
+- How to discover malicious intent from logs(?)
+
+
+== What to look for
+
+// How to tie API calls back to cluster users? Via Token?
+- Top API Server consumers
+- API requests by IP: high usage from a single IP may reveal a misbehaving user or misconfigured service
+- Recent API requests with users having cluster-admin privileges
```

Comment:
yes... actually, just retrieving a list of who has cluster-admin (or other high-level) privs is probably useful. and not as straightforward as you might think, though i guess for simplicity we can just check group membership e.g.:

```
I0907 17:24:13.550666       1 audit.go:133] 2017-09-07T17:24:13.550647939Z AUDIT: id="1a0ce9e6-b4cc-4afb-b10a-8769b3b21678" ip="172.18.6.149" method="DELETE" user="system:admin" groups="\"system:cluster-admins\"
I0907 17:24:13.553212       1 audit.go:48] 2017-09-07T17:24:13.553202441Z AUDIT: id="1a0ce9e6-b4cc-4afb-b10a-8769b3b21678" response="200"
```

(there are probably still other ways to grant privileges without putting the user in a group)